### ISSUE TITLE
Alternative palette for solids (bars and areas).

### ISSUE THREAD

**Issue by kavanagh** [ft-interactive/o-charts#66]
There is a an alternative sequence of colours for use on bars and areas. It has a more limited range of colours. For use when for whatever reason the pinky-browny-grey palette isn't appropriate.

@cnev177 can provide details of the palette.

**Comment by radiodario** [comment]
Have you seen #65 ? i added the colours specified on #45

**Comment by kavanagh** [comment]
Yes I saw that. It reminded me to log this issue. The alternative palette doesn't seem to be in that doc for some reason, it was in earlier versions.

Caroline can hopefully fill in the details.

**Comment by kavanagh** [comment]
@peter-mouland is this done?

**Comment by radiodario** [comment]
i think this is done AFAIK

**Comment by peter-mouland** [comment]
a pre-defined alternate palette isn't available. but users can easily code for different colours.

We could either : 
 * add 'alternateColours' as config to allow 2 sets of predefined colours  for easy swapping (i.e. monochrome). 
 * Or, we move this issue into nightingale and add UI to allow the user switch to set defined by nightingale

**Comment by kavanagh** [comment]
How about something like this:

```js

d3.select(svg)
    .data({
      title: 'Large tech companies last week',
      x: {
        series: 'timestamp'
      },
      y: {
        series: [
          {key: 'aapl', label: 'Apple', color: '#00ff00'},
          {key: 'goog', label: 'Google', color: '#ff0000'},
          {key: 'msft', label: 'Microsoft', color: '#0000ff'}
        ]
      },
      data: data
    })
    .call(oCharts.chart.line);
```

* Allow the developer to set a colour per series for maximum flexibility.
* If a `color` attribute is not set (or null) for a given y series then the default colors array is used.
* Allow devs to overwrite the global array of default colours too.

**Comment by peter-mouland** [comment]
sounds good
